en/publish/publish-wechatgame.md

```diff
@@ -1,114 +1,99 @@
-# 发布到微信小游戏平台
+# Publish to __WeChat__ Mini Game Platform
 
-早在微信官方正式发布小游戏平台之前，Cocos Creator 团队就和微信开发团队合作完成了 Cocos Creator 对微信小游戏平台的支持。在小游戏正式公布之后，我们也在第一时间发布了完美支持小游戏的 Cocos Creator v1.8。这篇文档的初衷也是希望帮助开发者们在第一时间以最顺畅的方式提交游戏到小游戏平台。
+![](./publish-__WeChat__game/preview.jpeg)
 
-![](./publish-wechatgame/preview.jpeg)
-
-这篇文档将会覆盖：
-
-- 小游戏的运行环境介绍
-- 如何使用 Cocos Creator 发布微信小游戏
-- 小游戏资源管理
-
-与这篇文档搭配的还有一段教学视频，可以结合视频中的操作来理解，更加直观。
+**Note:** there is also a teaching video that can be used alongside this document.
 
 {% raw %}
-<iframe frameborder="0" width="100%" height="440" src="https://v.qq.com/iframe/player.html?vid=c05255stri7&tiny=0&auto=0" allowfullscreen></iframe>
+<iframe frameborder="0" width="100%" height="440" src="https://v.qq.com/iframe/player.html?vid=c05255stri7&tiny=0&auto=0" allowfullscreen></iframe >
 {% endraw %}
 
-## 微信小游戏平台
-
-微信小游戏是微信小程序下的游戏产品平台，它不仅提供了强大的游戏能力，还和小程序一样，提供了大量的原生接口，比如支付，社交，文件系统，照片，NFC 等。相当于同时结合了 WEB 易于传播以及 Native 功能丰富的优势。
-
-小游戏的运行环境是小程序环境的扩展，基本思路也是封装必要的 WEB 接口提供给用户，尽可能追求和 WEB 同样的开发体验。小游戏在小程序环境的基础上提供了 WebGL 接口的封装，使得渲染能力和性能有了大幅度提升。不过由于这些接口都是微信团队通过自研的原生实现封装的，所以并不可以等同为浏览器环境。关于游戏方面，目前小游戏环境提供了 Canvas 以及 WebGL 的渲染接口，所以 Cocos Creator 引擎的两种渲染模式都是可以运行的，不过出于性能的考量，我们默认使用 WebGL 渲染游戏内容，强烈建议用户不要修改默认配置。
+## __WeChat__ Mini Game Platform
+__WeChat Mini Game__ is a game product platform used inside the __WeChat__ application. It not only provides powerful game capabilities, but also provides a large number of native interfaces, such as *payment*, *social*, *file system*, *photo*, *NFC*, and many more. It is equivalent to combining the advantages of the web for easy dissemination and rich native features.
 
-作为引擎方，为了尽可能简化开发者的工作量，我们为用户完成的主要工作包括：
+The running environment of the __WeChat Mini Game__ is an extension of the mini program environment. The basic idea is to provide the necessary web interface to the user, and pursue the same development experience as the web. The __WeChat Mini Game__ provides a wrapper around the WebGL interface based on the applet environment, which greatly improves the rendering capabilities and performance. However, since these interfaces are encapsulated by the __WeChat__ team they are not equivalent to the browser environment. Regarding the game aspect, the current __WeChat Mini Game__ environment provides the rendering interface of __Canvas__ and __WebGL__, so the two rendering modes of the __Cocos Creator__ engine can be run, but for performance reasons, we use __WebGL__ to render the game content by default, and users are strongly recommended. Do not modify the default configuration.
```

Comment:
1. applet environment -> mini program environment
2. I think the last sentence needs to use ","
and users are strongly recommended **, do** not modify the default configuration.

Reply:
1. fixed.

2. I modified a few sentences for better clarity. Sometimes I have to make sure I dont change the meaning and your pointers help.